udm-le.sh

```diff
@@ -10,22 +10,20 @@ DOCKER_VOLUMES="-v ${UDM_LE_PATH}/lego/:/var/lib/lego/"
 LEGO_ARGS="--dns ${DNS_PROVIDER} --email ${CERT_EMAIL} --key-type rsa2048"
 
 deploy_cert() {
-	CERT_IMPORT_CMD='java -jar /usr/lib/unifi/lib/ace.jar import_key_cert'
-	UBIOS_CERT_PATH='/mnt/data/unifi-os/unifi-core/config'
-	UNIFIOS_CERT_PATH='/data/unifi-core/config'
-
 	if [ "$(find -L "${UDM_LE_PATH}"/lego -type f -name "${CERT_NAME}".crt -mmin -5)" ]; then
 		echo 'New certificate was generated, time to deploy it'
 		# Controller certificate
 		cp -f ${UDM_LE_PATH}/lego/certificates/${CERT_NAME}.crt ${UBIOS_CERT_PATH}/unifi-core.crt
 		cp -f ${UDM_LE_PATH}/lego/certificates/${CERT_NAME}.key ${UBIOS_CERT_PATH}/unifi-core.key
 		chmod 644 ${UBIOS_CERT_PATH}/unifi-core.*
 
-		# Import the ertificate for the captive portal
-		podman exec -it unifi-os ${CERT_IMPORT_CMD} ${UNIFIOS_CERT_PATH}/unifi-core.key ${UNIFIOS_CERT_PATH}/unifi-core.crt
-
-		# This doesn't reboot your router, it just restarts the UnifiOS container
-		unifi-os restart
+		# Import the certificate for the captive portal
+		if [ "$ENABLE_CAPTIVE" == "yes" ]; then
+			podman exec -it unifi-os ${CERT_IMPORT_CMD} ${UNIFIOS_CERT_PATH}/unifi-core.key ${UNIFIOS_CERT_PATH}/unifi-core.crt
+	
+			# This doesn't reboot your router, it just restarts the UnifiOS container
```

Comment:
You're going to want to move the restart outside of this `if` otherwise if ENABLE_CAPTIVE isn't yes and there's a new cert it's not going to restart the container (which is still necessary outside of captive portal stuff).

Reply:
Ahh I didn't realize that was needed outside the captive addition, fixing.